test/antithesis/cp-combined/launch-cp

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+set -euo pipefail
+
+/usr/bin/zookeeper-server-start /etc/kafka/zookeeper.properties &
```

Comment:
I wonder how those commands work without the bazillion environment options they expect , which were set in the original Dockerfiles for each service?

Reply:
The APT packages mercifully ship with `.properties` files for each of these services that configure them to talk to one another on `localhost`. Unlike the Docker images, which have to be explicitly told about the other containers that exist.